clientd3d/d3drender.c

```diff
@@ -7012,8 +7029,15 @@ void D3DRenderObjectsDraw(d3d_render_pool_new *pPool, room_type *room,
 		// Nodes with a bound height adjust are part of other players' upper bodies.
 		if (pRNode->boundingHeightAdjust == 0)
 		{
+			// For objects in a similar position we increase the z depth
+			// to prevent z fighting at that location.
+			float binSize = 5.0f;

```

Comment:
We have FINENESS constants and others that I think you can reuse rather than introducing another one.